### ISSUE TITLE
DeepLabV3+ training settings

### ISSUE THREAD

**Issue by ErikStammes** [visinf/1-stage-wseg#5]
I know it's a little bit out of scope of the paper and this repository, but I was wondering which hyperparameters you used for training and evaluating DeepLabV3+ Xception to get such a performance increase (64.3 -> 67.3)?

Thanks a lot.

**Comment by arnike** [comment]
Hi @ErikStammes,

no problem, it's a fair question. Note that the mask IoU that we used for training is 66.9% (see Table 2).

We followed the [official setup](https://github.com/tensorflow/models/blob/master/research/deeplab/g3doc/pascal.md) and [the comments in the code](https://github.com/tensorflow/models/blob/master/research/deeplab/train.py). The only difference is that DeepLab is originally trained in two steps: 30K iterations on VOC+SBD, then another 30K iterations on VOC only. This is because the ground truth for SBD is much more coarse than that of VOC. With our pseudo labels we trained only for the first 30K iterations, because in our case the mask accuracy between VOC and SBD is uniform. We did not fine-tune the training schedule, so perhaps more can be gained there.


I recall that the official details for training deeplabv3+ are a bit disguised and scattered. So, I provide the key details here for a reference:

### Xception
```
--model_variant="xception_65"
--atrous_rates=6
--atrous_rates=12
--atrous_rates=18
--output_stride=16
--decoder_output_stride=4
--train_crop_size=513
--train_crop_size=513
--train_batch_size=16
--base_learning_rate=0.007
--training_number_of_steps=30000
--fine_tune_batch_norm=true
--num_clones=4
```

### ResNet-101
```
--model_variant="resnet_v1_101_beta"
--atrous_rates=6
--atrous_rates=12
--atrous_rates=18
--output_stride=16
--decoder_output_stride=4
--train_crop_size=513
--train_crop_size=513
--train_batch_size=16
--base_learning_rate=0.007
--weight_decay=0.0001
--training_number_of_steps=30000
--fine_tune_batch_norm=true
--num_clones=4
```

**Comment by ErikStammes** [comment]
Thanks @arnike, that's exactly what I was looking for.
